pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -394,6 +394,10 @@ func (gce *GCECloud) EnsureTCPLoadBalancer(name, region string, externalIP net.I
 		PortRange:  fmt.Sprintf("%d-%d", minPort, maxPort),
 		Target:     gce.targetPoolURL(name, region),
 	}
+	if loadBalancerIP != nil {
+		req.IPAddress = loadBalancerIP.String()
```

Comment:
Aha, thanks - I think I finally understand now.  The OpenStack plugin has always (perhaps mistakenly) implemented externalIP exactly as you're trying to do with this change, hence my confusion.  On the upside, that means the plugin should just continue to work once we formalise that interpretation :)